covid_graphs/covid_graphs/country_graph.py

```diff
@@ -83,28 +85,35 @@ def create_country_figure(self):
                      y0=0,
                      y1=1,
                      fillcolor="LightGreen",
-                     opacity=0.5,
+                     opacity=0.3,
                      layer="below",
                      line_width=0))
         except ValueError:
             pass
 
-        layout = Layout(title=f"Active cases in {self.name}",
-                        xaxis=dict(
-                            autorange=True,
-                            title=f'Day [starting at the {self.min_case_count}th case]',
-                        ),
-                        yaxis=dict(autorange=True,
-                                   title=f'COVID-19 active cases in {self.name}',
-                                   tickformat='.0f'),
-                        height=700,
-                        shapes=shapes,
-                        hovermode='x',
-                        font=dict(size=20),
-                        legend=dict(x=0.01, y=0.99, borderwidth=1))
+        layout = Layout(
+            title=f"Active cases in {self.name}",
+            xaxis=dict(
+                autorange=True,
+                title=f'Day [starting at the {self.min_case_count}th case]',
+                showgrid=False,
+            ),
+            yaxis=dict(
+                autorange=True,
+                tickformat='.0f',
```

Comment:
Interesting, does this print e.g. "80k" as in your screenshot? From the docs I thought it should be `"~s"`: https://github.com/d3/d3-format/blob/master/README.md#locale_format
(Also I like the "80k" format.)

Reply:
80k is confusing to general public.

I'd prefer something along
```
d3.format(",.2r")(4223);  // grouped thousands with two significant digits, "4,200"
```

It has a divider (a comma) and does some rounding. Though I'd skip the rounding.